lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -100,6 +100,11 @@ Function Choco-IsInstalled
         [string]$package
     )
 
+   if ($package -like "all")
```

Comment:
Sorry for not getting back to this but appreciate the effors @basictheprogram. `-eq` is case insensitive while `-ceq` is the case sensitive version. We merged `-eq` into devel just recently.